### PR TITLE
Delta 1.2 blog and Updates to main page highlights

### PR DESCRIPTION
**Pull request by vinijaiswal**
- Delta 1.2 blog released
- Featured section updated with latest 4 community engagements.

### CODE REVIEW

**Comment by vinijaiswal** [comment]
Signed-off-by: Vini Jaiswal <vini.js1303@gmail.com>
@dennyglee Can you please review? Thank you!

**Review by dennyglee — CHANGES_REQUESTED** [review]
Great blog - please update per requested changes - thanks!

File: src/blog/2022-05-05-delta-lake-1-2-released/index.mdx [hunk 1]
```diff
@@ -0,0 +1,215 @@
...
+## Summary
+
+Delta Lake is quickly becoming the de facto, open-source storage framework for companies building transformative data platforms on the Lakehouse architecture. With data volumes and complexity only increasing, it’s important to ingest data at massive scale faster and reliably. With Delta Lake 1.2, developers can take full advantage of the performance optimizations and the operational efficiencies designed to boost productivity.
+
+## What’s next
+
+We are already gearing up for many new features in the next release of Delta Lake. You can track all the upcoming releases and planned features in [GitHub milestones](https://github.com/delta-io/delta/milestones).
+
+> **Interested in the open-source Delta Lake?**
+> Visit [Delta Lake](https://github.com/delta-io/delta) to learn more, you can join the Delta Lake community via [Slack](http://dbricks.co/delta-users-slack) and [Google Group](https://groups.google.com/forum/#!forum/delta-users).
```
Comment (dennyglee) [inline src/blog/2022-05-05-delta-lake-1-2-released/index.mdx:211]: Please change the link from dbricks.co/delta-users-slack to go.delta.io/delta-users

**Comment by dennyglee** [comment]
Thanks @vinijaiswal - there are four issues remaining - could you please address them and we'll get this closed out.  Thanks!

File: src/blog/2022-05-05-delta-lake-1-2-released/index.mdx [hunk 2]
```diff
@@ -0,0 +1,215 @@
...
+<b>
+  <center>
+    Fig.1. Better automatic data compaction for faster query performance.
+  </center>
+</b>
+<b></b>
+&nbsp;
+
+Through the reduction of the number of data files being ingested, compacting of small files into larger files in a Delta table can improve read latency as a direct result of reducing metadata size and per-file overheads such as file-open overhead and file-close overhead.
+-- Optimizes Delta Lake table
```
Comment (dennyglee) [inline src/blog/2022-05-05-delta-lake-1-2-released/index.mdx:56]: Lines 56 and 57 should be included in the lines 60 and 61.  That is lines 61. should be 
```sql
OPTIMIZE delta_table_name;                -- Optimizes Delta Lake Table
OPTIMIZE delta.`/path/to/delta/table`;    -- Optimizes the path-based Delta Lake table
```

**Review by dennyglee — CHANGES_REQUESTED** [review]
Could you please undo the `package-lock.json` and validate locally that the original version still works?

File: src/blog/2022-05-05-delta-lake-1-2-released/index.mdx [hunk 3]
```diff
@@ -0,0 +1,213 @@
...
+## Summary
+
+Delta Lake is quickly becoming the de facto, open-source storage framework for companies building transformative data platforms on the Lakehouse architecture. With data volumes and complexity only increasing, it’s important to ingest data at massive scale faster and reliably. With Delta Lake 1.2, developers can take full advantage of the performance optimizations and the operational efficiencies designed to boost productivity.
+
+## What’s next
+
+We are already gearing up for many new features in the next release of Delta Lake. You can track all the upcoming releases and planned features in [GitHub milestones](https://github.com/delta-io/delta/milestones).
+
+> **Interested in the open-source Delta Lake?**
+> Visit [Delta Lake](https://github.com/delta-io/delta) to learn more, you can join the Delta Lake community via [Slack](go.delta.io/delta-users) and [Google Group](https://groups.google.com/forum/#!forum/delta-users).
```
Comment (dennyglee) [inline src/blog/2022-05-05-delta-lake-1-2-released/index.mdx:209]: Please change `go.delta.io/delta-users` to `https://go.delta.io/delta-users`

**Review by dennyglee — CHANGES_REQUESTED** [review]
Please revert the package-lock.json to a previous version of the file - thanks!

File: src/blog/2022-05-05-delta-lake-1-2-released/index.mdx (hunk 3 above)
Comment (dennyglee) [reply src/blog/2022-05-05-delta-lake-1-2-released/index.mdx]: thanks!

**Review by dennyglee — APPROVED** [review]
LGTM!

**Pull request merged**